frontend/spec/selenium/spec/digital_objects_spec.rb

```diff
@@ -1,274 +0,0 @@
-# frozen_string_literal: true
-
-require_relative '../spec_helper'
-
-describe 'Digital Objects' do
-  before(:all) do
-    @repo = create(:repo, repo_code: "do_test_#{Time.now.to_i}")
-    set_repo @repo
-
-    @do = create(:digital_object)
-    @cl = create(:classification)
-
-    @alt_do_title = 'File version alt text test'
-    @alt_doc_title = 'Child of file version alt text test'
-    alt_uri = 'https://www.archivesspace.org/demos/Congreave%20E-2/ms292_003_page002.jpg'
-    alt_format = 'jpeg'
-    @alt_caption = 'This is the caption'
-
-    @alt_do = create(
-      :digital_object,
-      title: @alt_do_title,
-      file_versions: [
-        {
-          file_uri: alt_uri,
-          file_format_name: alt_format,
-          caption: @alt_caption
-        },
-        {
-          file_uri: alt_uri,
-          file_format_name: alt_format
-        }
-      ]
-    )
-    @alt_doc = create(
-      :digital_object_component,
-      digital_object: { ref: @alt_do.uri },
-      title: @alt_doc_title,
-      file_versions: [
-        {
-          file_uri: alt_uri,
-          file_format_name: alt_format,
-          caption: @alt_caption
-        },
-        {
-          file_uri: alt_uri,
-          file_format_name: alt_format
-        }
-      ]
-    )
-
-    run_all_indexers
-
-    @do_child1 = create(:digital_object_component, digital_object: { ref: @do.uri })
-    @do_child2 = create(:digital_object_component, digital_object: { ref: @do.uri })
-
-    @do2 = create(:digital_object)
-
-    user = create_user(@repo => ['repository-archivists'])
-    @driver = Driver.get.login_to_repo(user, @repo)
-  end
-
-  after(:all) do
-    @driver ? @driver.quit : next
-  end
-
-  it 'reports errors and warnings when creating an invalid Digital Object' do
-    @driver.find_element(:link, 'Create').click
-    @driver.click_and_wait_until_gone(:link, 'Digital Object')
-    @driver.find_hidden_element(:css, '#digital_object_title_').wait_for_class('initialised')
-    @driver.execute_script("$('#digital_object_title_').data('CodeMirror').setValue('')")
-
-    @driver.click_and_wait_until_gone(css: "form#new_digital_object button[type='submit']")
-
-    @driver.find_element_with_text('//div[contains(@class, "error")]', /Identifier - Property is required but was missing/)
-
-    # cancel those changes (shows a new form)
-    @driver.click_and_wait_until_gone(:css, 'a.btn.btn-cancel')
-
-    # and jump back home so we can start again
-    @driver.go_home
-  end
-
-  digital_object_title = 'Pony Express Digital Image'
-
-  it 'can create a digital_object with one file version' do
```

Comment:
I notice this test isn't carried over. Perhaps it's ok since a DO w/ multiple file versions is created in the following test?

Reply:
exactly!